about.html

```diff
@@ -0,0 +1,117 @@
+        <script
+          src="https://kit.fontawesome.com/b21ce73c3c.js"
+          crossorigin="anonymous"
+        ></script>
+    
+       <!-- Google fonts ------- -->
+       <link rel="preconnect" href="https://fonts.googleapis.com">
+       <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+       <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap" rel="stylesheet">
+       
+       
+    
+        <!-- local style links-->
+        <link rel="icon" href="images/logo.png" />
+         <link rel="stylesheet" href="./css/about.css">
+        
+        <title> Global Streaming </title>
+    </head>
+<body>
+    <header class="header">
+        <!-- top menu start-->
+        <div class="top-menu">
+            <ul class="top-menu-ul">
+                <li class="a"> <a href=""> <span class="fa fa-facebook"></span>
+                </a></li>
+                <li class="a"> <a href=""> <span class="fa fa-twitter"></span></a></li>
+                <li class="a"> <a href=""> English</a></li>
+                <li class="a"> <a href=""> Mypage</a></li>
+                <li class="a"> <a href=""> Logout</a></li>
+            </ul>
+        </div>
+         <!-- menu start-->
+        <nav class="main-menu">
+           <a href="index.html"> <img src="./images/logo.png" alt="logo"> <p> Global <span> Streaming</span> </p></a>
+           <span class="icon">
+            <span class="fa fa-bars"   aria-hidden="true"></span>
+            <span class="fa fa-times"   aria-hidden="true"></span>
+           </span>
+            <ul class="main-menu-ul">
+                
+                <li> <a href="about.html" class="active"> About </a></li>
+                <li><a href="index.html" > Home</a></li>
+                <li> <a href="#"> Live_Streaming </a></li>
+                <li class="disapper"> <a href="index.html#event-categories"> Event </a></li>
+                <li class="disapper"><a  href="#event">See recent</a></li>
+                <li class="disapper"> <a href="#">  Logout </a></li>
+            </ul>
+        </nav>
+    </header>
+<main>
+    <section class="descrption">
+    <h2>GLOBAL . STREAMING</h2>
+    <h2> BEST LIVE STREAMING WEBSITE -2022</h2>
+    <div>
+        <p>Global Streaming is a live-streaming website that allow users to record, 
+            broadcast video and stream it live simultaneously. Also it allows users to broadcast
+             live recordings of videos and audio. Others who have access to the Global Live video streaming,
+              have fueled influencer marketing, entertainment, business marketing, and the way people interact 
+              over the internet.
+             Global Streaming is the best website for sharing and streaming live event in 2022.</p>
+    </div>
+    <p>Please contact us via email for feedback or any other inquiry about Global Streaming</p>
+    <i>globalstreaming2022@gmail.com</i>
+    </section>
```

Comment:
- [ ] Please know that you should follow the design as close as possible, this is how your section looks:
  ![image](https://user-images.githubusercontent.com/20567503/198015756-a7f3eacb-1a85-4596-bd26-abc6bba20d01.png)
  And this is how it should look:
  ![image](https://user-images.githubusercontent.com/20567503/198015807-a795e038-e47f-4248-9ce9-dc0beeac786b.png)
  Kindly fix the design. 😄